app/src/composables/use-permissions/collection/lib/is-action-allowed.ts

```diff
@@ -0,0 +1,15 @@
+import { usePermissionsStore } from '@/stores/permissions';
+import { computed, unref } from 'vue';
+import { Collection } from '../../types';
+
+export const isActionAllowed = (collection: Collection, action: 'create' | 'read' | 'update' | 'delete') => {
```

Comment:
```suggestion
export const isActionAllowed = (collection: Collection, action: 'create' | 'read' | 'update' | 'delete' | 'share') => {
```

Reply:
Hmm I'd only list the actions that are actually used, `share` isn't one of those. If we need that later on we can still expand the list.

Reply:
Agreed, looks like I confused that with the item equivalent.